Resources/Prototypes/_LostParadise/Entities/Clothing/Head/softsuits-helmets.yml

```
@@ -0,0 +1,172 @@
+    energy: 2
+    mask: /Textures/Effects/LightMasks/cone.png
+    autoRot: true
+    netsync: false
+
+- type: entity
+  parent: ClothingHeadEVAHelmetWithLightBase
+  id: ClothingHeadEVAHelmetEngineer
+  name: engineer EVA helmet
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)